fiftyone/core/stages.py

```diff
@@ -897,6 +897,78 @@ def validate(self, sample_collection):
         fova.validate_video_collection(sample_collection)
 
 
+class ExcludeGroups(ViewStage):
+    """Excludes the groups with the given IDs from a grouped collection.
+
+    Examples::
+
+        import fiftyone as fo
+        import fiftyone.zoo as foz
+
+        dataset = foz.load_zoo_dataset("quickstart-groups")
+
+        #
+        # Select some specific groups by ID
```

Comment:
nit: update docstrings to say "exclude" rather than "select" in a few places here

Reply:
fixed